docs/tutorials/services/color-detection-scuttle.md

```diff
@@ -315,81 +319,86 @@ You could also write some code with a Viam SDK to [make your rover move in a squ
 
 ```python {class="line-numbers linkable-line-numbers"}
 import asyncio
-​
+
 from viam.robot.client import RobotClient
 from viam.rpc.dial import Credentials, DialOptions
 from viam.services.vision import VisionServiceClient
 from viam.services.vision import Detection
 from viam.components.camera import Camera
 from viam.components.base import Base
-​
-​
+
+
 async def connect():
-   creds = Credentials(
+    creds = Credentials(
         type="robot-location-secret",
-        payload="[PLEASE ADD YOUR SECRET HERE. YOU CAN FIND THIS ON THE CODE SAMPLE TAB]")
-   opts = RobotClient.Options(
+        payload="[PLEASE ADD YOUR SECRET HERE. "
+                "YOU CAN FIND THIS ON THE CODE SAMPLE TAB]")
```

Comment:
I wonder if we should exclude this -- users will think the creds are two parts (like our [org API keys](https://docs.viam.com/manage/cli/#create-an-organization-api-key) are, for example, easy to confuse here!!). 

I would vote for a single line,  but shorter text:

```suggestion
        payload="<location_secret>")
```

Or `payload='YOUR LOCATION SECRET'  # The location secret`
or `location-secret`,
or ...

Like on many other code samples. Same with the robot address just below.